netbox/project-static/styles/netbox.scss

```diff
@@ -167,6 +167,13 @@ table td > .progress {
   }
 }
 
+.alert {
+  code {
+    color: $gray-600;
+    margin: 0 1em 0 1em;
```

Comment:
IMO we should keep the `mx-1` CSS class in place on the element for this, just to make it easier to adjust if need be in the future.